core/server/data/migrations/versions/3.38/05-add-emails-track-opens-column.js

```diff
@@ -0,0 +1,28 @@
+const {createColumnMigration, addColumn, dropColumn} = require('../../../schema').commands;
+
+module.exports = {
+    config: {
```

Comment:
No biggie but we don't need this as DDL operations can't be performed within transactions! Doesn't matter if we leave it though :relaxed: